.github/workflows/pr-ci.yml

```
@@ -456,9 +520,10 @@ jobs:
             requires-secret: false
 
     runs-on: ${{ matrix.runner }}
-    timeout-minutes: 45
+    timeout-minutes: ${{ matrix.job-timeout-minutes || 45 }}
     steps:
       - uses: jlumbroso/free-disk-space@54081f138730dfa15788a46383842cd2f914a1be # v1.3.1
+        background: true
```

actionlint (1.7.12)

[error] unexpected key "background" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)